server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/lib/AbstractStateMailetProcessor.java

```diff
@@ -429,8 +429,8 @@ public void service(Mail mail) {
                 // error processor. That is currently the
                 // normal situation for James, and the message
                 // will show up in the error store.
-                String warnBuffer = "Message " + mail.getName() + " reached the end of this processor, and is automatically deleted.  This may indicate a configuration error.";
-                logger.warn(warnBuffer.toString());
+                logger.warn("Message " + mail.getName() + " reached the end of this processor, and is automatically deleted." +
```

Comment:
`logger.warn("Message {} reached the end of this processor, and is automatically deleted. This may indcate a configuration error.", mail.getName())`